docs/configuration.md

```diff
@@ -209,10 +211,10 @@ Apart from these, the following properties are also available, and may be useful
   <td>(none)</td>
   <td>
     Set a special library path to use when launching the driver JVM.
-    
+
```

Comment:
@srowen I gave this some thought. I think we can both support the new interface and maintain compatibility.

Put simply, throughout the code I use `Utils.timeStringToMs` or `Utils.timeStringToS` for all specified constants as I do now. The only thing thing that needs to change is the default behavior for these. E.g. if the parameter is read in the code with `timeStringToMs` then if the format is unspecified (600 vs 600 ms) then the correct modifier will be applied. Otherwise, whichever unit is provided is used.

This will allow us to introduce the new interface, update all usage of constants throughout the code to clarify units, and to support old usage. It will also cut down on the number of code changes since documentation can be left as is.